store/src/main/java/randomStorePopulator/RandomStorePopulator.java

```diff
@@ -0,0 +1,43 @@
+package randomStorePopulator;
+
+import com.github.javafaker.Faker;
+import product.Product;
+import store.Store;
+
+import java.util.ArrayList;
+
+public class RandomStorePopulator {
+
+    private final Faker faker = new Faker();
+
+    private String getRandomProductName() {
+        return faker.food().ingredient();
```

Comment:
Simple but not acceptable :) . You have different categories (Hat, Tshirt, Hat)  and logically names from food are not correct for them.